app/uk/gov/hmrc/pegaproofofconceptfrontend/controllers/InputController.scala

```diff
@@ -49,7 +52,10 @@ class InputController @Inject() (
       },
       (validFormData: StringInputForm) => {
         logger.info(s"SUBMITTED STRING: '${validFormData.string}' TO PEGA")
-        Future.successful(Redirect(uk.gov.hmrc.pegaproofofconceptfrontend.controllers.routes.InputController.getStringInput))
+        pegaProxyConnector.submitPayloadToProxy(Payload.fromStringInputForm(validFormData)).map {
+          case response if response.status === 200 => Ok(views.fakePegaPage())
+          case _                                   => Redirect(uk.gov.hmrc.pegaproofofconceptfrontend.controllers.routes.ProblemWithTheServiceController.problemWithTheService)
```

Comment:
you may not necessarily need a separate page for this - you could just throw an exception here and it should just show the default tech difficulties page

Reply:
isn't the standard techincal difficulties page given to you by the HMRC libraries actively discouraged upon by govuk design system, they prefer you create your own version of this page and use it https://design-system.service.gov.uk/patterns/problem-with-the-service-pages/

Reply:
haven't seen that before but it's good to know about 👍 In this case since it's a throwaway service that is never gonna make it's way in front of users I think we're ok with the standard one - less code to maintain is easier to manage